### PR TITLE
Acceptance test 17.2.10

### PR DESCRIPTION
**Pull request by gitshrikant**
Successfully passed all test cases.

### CODE REVIEW

File: avi/provider_test.go [hunk 1]
```diff
@@ -78,32 +78,25 @@ func TestProvider(t *testing.T) {
...
 	}
+
 	_, err := clients.NewAviClient(
 		config.Controller, config.Username,
 		session.SetPassword(config.Password),
 		session.SetTenant(config.Tenant),
+		session.SetVersion(config.Version),
 		session.SetInsecure)
 
 	if err != nil {
```
Comment (Rishabh-Baranwal) [inline avi/provider_test.go]: Revert this file and set the version to 17.2.8 if not provided

Comment (gitshrikant) [reply avi/provider_test.go]: Reverted and set version to 17.2.8.

File: avi/provider.go [hunk 2]
```diff
@@ -26,13 +26,13 @@ func Provider() terraform.ResourceProvider {
 			},
 			"avi_controller": &schema.Schema{
 				Type:        schema.TypeString,
-				Optional:    true,
+				Required:    true,
```
Comment (grastogi23) [inline avi/provider.go]: pl. fix the template and regenerate

File: avi/provider.go [hunk 3]
```diff
@@ -208,7 +210,7 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		session.SetVersion(config.Version),
 		session.SetInsecure)
 
-	log.Println("Avi Client created for user %v tenant %v version %v",
+	log.Printf("Avi Client created for user %v tenant %v version %v",
```
Comment (grastogi23) [inline avi/provider.go]: this should be println

**Pull request merged**